.env.example

```diff
@@ -1 +1,2 @@
 PORT=3000
+REDISTOGO_URL=redis://localhost:6379/
```

Comment:
I could see that going a couple of ways. Either what you suggest, or to remove the default entirely and have it blow up because it's mis-configured.

That's really only used in production right? In development mode we run Rails and ember server separately so we don't use redis for serving the ember index. Is that right? If so, I would lean to making it blow up so that we know right away that it's not configured.